scoring_engine/engine/checks/rdp.py

```diff
@@ -0,0 +1,16 @@
+from scoring_engine.engine.basic_check import BasicCheck
+
+
+class RDPCheck(BasicCheck):
+    required_properties = []
+
+    CMD = 'xfreerdp --ignore-certificate --authonly -u {0} -p {1} {2}:{3}'
```

Comment:
I need to update this so that we print a success message is the error code == 0, so I may have to create another file and add it to checks_bin